### DISCUSSION TITLE
Conflict with readline shortcuts?

### DISCUSSION THREAD

**Discussion by mvsrgc** [mrjones2014/smart-splits.nvim#136] [Q&A]
Smart-splits uses Ctrl-l to move the cursor and suggests that tmux users bind Ctrl-l in tmux to move the cursor, but this conflicts with readline's Ctrl-l to clear the screen?

**Comment by mrjones2014** [comment]
Yeah, I don't personally use ctrl+l readline shortcut so it's fine with me. But the plugin doesn't set any keymaps. You can use whatever you want. If you don't want the conflict then don't use ctrl+l.
